frontend/src/components/shared/TooltipIcon/TooltipIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from "react"
+import Tooltip, { Placement } from "components/shared/Tooltip"
+import { HelpCircle as HelpCircleIcon } from "react-feather"
+import StreamlitMarkdown from "components/shared/StreamlitMarkdown"
+
+export interface TooltipIconProps {
+  placement?: Placement
+  iconSize?: string
+  content: string
+}
+
+function TooltipIcon({
+  placement = Placement.AUTO,
+  iconSize = "16",
+  content,
+}: TooltipIconProps): ReactElement {
+  return (
+    <div className="stTooltipIcon">
+      <Tooltip
+        content={<StreamlitMarkdown source={content} allowHTML />}
+        placement={placement}
+      >
+        <HelpCircleIcon size={iconSize} />
+      </Tooltip>
+    </div>
+  )
+}
+
+export { Placement }
```

Comment:
I don't think exporting in multiple places is a good idea. Can we import it from the original place where we need it?